.env.example

```
@@ -7,10 +7,12 @@ HYPERCODE_JWT_SECRET=your_jwt_secret_here    # JWT secret for auth
 HYPERCODE_MEMORY_KEY=your_memory_encryption_key_here
 
-# Root path for all bound volumes (see docker-compose.yml volumes)
-# Example (Windows): HC_DATA_ROOT=C:\HyperCodeData
-# Example (Linux):   HC_DATA_ROOT=/opt/hypercode-data
-HC_DATA_ROOT=/absolute/path/to/hypercode-data
+# Root path for all bound volumes (see docker-compose.yml volumes).
+# Windows: use forward slashes. No quotes. Spaces in path are fine.
+#   Example: HC_DATA_ROOT=H:/HyperStation zone/HyperCode/HyperCodeData
+# Linux/macOS:
+#   Example: HC_DATA_ROOT=/opt/hypercode-data
+HC_DATA_ROOT=H:/HyperStation zone/HyperCode/HyperCodeData
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)